app/src/main/res/layout/activity_main.xml

```diff
@@ -22,7 +22,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         app:layout_insetEdge="bottom"
-        android:visibility="gone"
+        android:visibility="visible"
```

Comment:
Ensures that the bottom nav is shown by default. It is explicitly hidden in the three welcome fragments.